types/types.ts

```diff
@@ -1,8 +1,10 @@
 import { Zora20210101 } from './zora/20210101';
+import { Zora20210604 } from './zora/20210604';
 import { Catalog20210202 } from './catalog/20210202';
 import { Amulet20210221 } from './amulet/20210221';
 
 export { Zora20210101 }
+export { Zora20210604 }
 export { Catalog20210202 }
 export { Amulet20210221 }
-export type MetadataLike = Zora20210101 | Catalog20210202 | Amulet20210221 ;
+export type MetadataLike = Zora20210101 | Zora20210604 | Catalog20210202 | Amulet20210221;
```

Comment:
Note: Do we need to update anything on zora.co to support this?
I imagine we can add in a feature to support this new metadata when minting.